templates/package/content/debian.tmpl

```diff
@@ -4,8 +4,8 @@
 		<div class="ui form">
 			<div class="field">
 				<label>{{svg "octicon-terminal"}} {{.locale.Tr "packages.debian.registry"}}</label>
-				<div class="markup"><pre class="code-block"><code>curl {{AppUrl}}api/packages/{{$.PackageDescriptor.Owner.Name}}/debian/repository.key | sudo apt-key add -
-echo "deb {{AppUrl}}api/packages/{{$.PackageDescriptor.Owner.Name}}/debian &lt;distribution&gt; &lt;component&gt;" >> /etc/apt/sources.list
+				<div class="markup"><pre class="code-block"><code>sudo curl {{AppUrl}}api/packages/{{$.PackageDescriptor.Owner.Name}}/debian/repository.key -o /etc/apt/trusted.gpg.d/gitea-{{$.PackageDescriptor.Owner.Name}}.asc
```

Comment:
I changed this because there isn't much display space at that page:
![grafik](https://user-images.githubusercontent.com/1666336/218270501-52107c6d-d401-4634-ab8e-11480d52779e.png)
But sure, I can change it back.